app/assets/stylesheets/pages/about.scss

```diff
@@ -32,14 +47,14 @@
   @extend .center;
 
   p.mission {
-    font-size: 1.5em;
+    font-weight: bold;
+    font-size: 1.2em;
+    color: #477DCA;
```

Comment:
Color literals like `#477DCA` should only be used in variable declarations; they should be referred to via variable everywhere else.